sites/upsun/src/manage-resources/resource-init.md

```diff
@@ -397,18 +397,15 @@ title=In the Console
 
 When you [restore a backup](/environments/restore.md) using the Console, it is restored to your current environment.
 
-Each container already running on your current environment keeps its existing resources.
-Therefore, you don't need to specify a resource initialization strategy. 
+Containers are restored using the `backup` strategy, meaning:
+- The resources of every container running on the environment are reverted to their original state when the backup was taken.
```

Comment:
This is not because the `backup` strategy is used.

So I would first explain above the default behaviour, that we restore resources to their original state when the backup was taken.

But that you can also use the resource initialization strategy to define how new containers will be restored, knowing that in the case of a backup restore, there is only 2 cases when new containers are initialized => 1) when you restore to a new environment, in that case, all containers are new, or 2) when you restore to an environment where you have removed some containers since the backup was taken, in that case, these removed containers will be restored using the resource initialization strategy.

Reply:
I was also thinking that we might want to add a short sentence about this in these 2 pages:
https://docs.upsun.com/environments/restore.html#2-restore-from-a-backup
https://docs.upsun.com/environments/backup.html